pages/success.tsx

```diff
@@ -240,14 +246,25 @@ export default function Success(props: inferSSRProps<typeof getServerSideProps>)
   );
 }
 
-export async function getServerSideProps(context) {
+export async function getServerSideProps(context: { query: { type: string } }) {
   const typeId = parseInt(asStringOrNull(context.query.type) ?? "");
   if (isNaN(typeId)) {
     return {
       notFound: true,
     };
   }
-  const eventType: EventType = await prisma.eventType.findUnique({
+  interface returnEventType {
+    users: { hideBranding: boolean; plan: UserPlan; name: string | null; theme: string | null }[];
+    id: number;
+    title: string;
+    description: string | null;
+    length: number;
+    userId: number | null;
+    eventName: string | null;
+    requiresConfirmation: boolean;
+    team: {} | null; // todo: make an interface / type for team
+  }
+  const eventType: returnEventType | null = await prisma.eventType.findUnique({
```

Comment:
This is inferred so shouldn't be needed

```tsx
  const eventType = await prisma.eventType.findUnique({
```

Reply:
then follow-up with 

```ts
const eventTypeWithUsers = {
  ...eventType,
  users: !eventType.users.length && eventType.userId ? await prisma.user.findUnique({...}) : eventType.users`
}
```

Reply:
Or maybe that second part isn't needed, we can probably just `push()` it like we do now - the type inference should already guarantee that it'shas all the same fields